src/main/resources/db/reference/modernised_darts_schema.sql

```diff
@@ -1201,6 +1244,7 @@ CREATE TABLE user_account
 (usr_id                      INTEGER                       NOT NULL
 ,dm_user_s_object_id         CHARACTER VARYING(16)
 ,user_name                   CHARACTER VARYING             NOT NULL
+,user_full_name              CHARACTER VARYING             NOT NULL

```

Comment:
I thought we were treating `user_name` to be a full name anyway previously?
I'm wondering, what's the expected difference between the 2 columns now